net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml

```diff
@@ -16,9 +16,6 @@
                     <ValidationMessage>Should be a string between 1 and 32 characters. Allowed characters are 0-9, a-z, and A-Z</ValidationMessage>
                 </name>
                 <instance type="AutoNumberField">
-                    <MinimumValue>0</MinimumValue>
```

Comment:
Hopefully -1 isn’t valid input?

Reply:
Its an AutoNumberField without any way to edit via UI, shall Min stay here?

Reply:
Maybe @AdSchellevis knows by memory, not at a computer at the moment.

Reply:
@mimugmail you should be safe, 0 is the default :)

https://github.com/opnsense/core/blob/af587e812a7b8d96cf3fd84869dc7e00bbfe09b3/src/opnsense/mvc/app/models/OPNsense/Base/FieldTypes/AutoNumberField.php#L71